ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -228,3 +228,76 @@ Open Dashboard Settings
     END
     Wait For RHODS Dashboard To Load    expected_page=${exp_page_title}
     ...    wait_for_cards=${FALSE}
+
+
+Get Checkbox State Of Multi Mode Serving Platforms
+    [Documentation]    Checks if  Multi model serving platform is enabled/checked in Cluster setting
+    ${is_checked} =  Run Keyword And Return Status  Checkbox Should Be Selected  ${MULTI_MODE_SERVING_CHECK_BOX}
+    [Return]  ${is_checked}
+
+Get Checkbox State Of Single Mode Serving Platforms
+    [Documentation]    Checks if  Single model serving platform is enabled/checked in Cluster setting
+    ${is_checked} =  Run Keyword And Return Status  Checkbox Should Be Selected  ${SINGLE_MODE_SERVING_CHECK_BOX}
+    [Return]  ${is_checked}
+
+Click CheckBox Single Mode Serving Platforms
+    [Documentation]    Enables Single model serving platform
+    ${status}=  Get Checkbox State Of Single Mode serving platforms
+    IF  "${status}"=="False"
+        Select Checkbox   ${SINGLE_MODE_SERVING_CHECK_BOX}
+    END
+
+Click CheckBox Multi Mode Serving Platforms
+    [Documentation]    Enables Multi Mode serving platforms
+    ${status}=  Get Checkbox State Of Multi Mode serving platforms
+    IF  "${status}"=="False"
+        Select Checkbox   ${MULTI_MODE_SERVING_CHECK_BOX}
+    END
+
+UnClick CheckBox Single Mode Serving Platforms
+    [Documentation]    Disable checkbox Single Mode serving platforms
+    ${status}=  Get Checkbox State Of Single Mode serving platforms
+    IF  "${status}"=="True"
+        Unselect Checkbox   ${SINGLE_MODE_SERVING_CHECK_BOX}
+    END
+
+UnClick CheckBox Multi Mode Serving Platforms
+    [Documentation]    Disable checkbox Multi Mode serving platforms
+    ${status}=  Get Checkbox State Of Single Mode serving platforms
+    IF  "${status}"=="True"
+        Unselect Checkbox   ${MULTI_MODE_SERVING_CHECK_BOX}
+    END
+
+Click Both Model Serving Platforms
+    [Documentation]    Enables Multi Mode and Single Mode serving platforms
+    Click CheckBox Single Mode Serving Platforms
+    Click CheckBox Multi Mode Serving Platforms
+
+UnClick Both Model Serving Platforms
+    [Documentation]    Disbale Multi Mode and Single Mode serving platforms
+    UnClick CheckBox Single Mode Serving Platforms
+    UnClick CheckBox Multi Mode Serving Platforms
+
+Check IF Single Mode Model Serving Is Disabled From DSC And Cluster Setting
+    [Documentation]    Check single mode model should be disabled from DSC and dashboard
+    Component Should Not Be Enabled  kserve
+    ${status}   Get Checkbox State Of Single Mode Serving Platforms
```

Comment:
could you also check that the checkbox is disabled/greyed out?

Reply:
sure ill try to do it